de.json

```
@@ -706,11 +706,11 @@
         "CameraControl.Settings.AimInFrontOfHead" : "Ausrichtung auf die Stirn",
         "CameraControl.Settings.HideCamera" : "Kamera unsichtbar",
         "CameraControl.Settings.ForceEyesOnCamera" : "Fokussiere Augen auf Kamera",
-        "CameraControl.Settings.HideBadges" : "Abzeichen unsichtbar",
-        "CameraControl.Settings.HideLasers" : "Laser unsichtbar",
+        "CameraControl.Settings.HideBadges" : "Namen ausblenden",
+        "CameraControl.Settings.HideLasers" : "Laser ausblenden",
         "CameraControl.Settings.ShowFrustum" : "Zeige Frustum",
         "CameraControl.Settings.PhotoTimer" : "Foto-Selbstauslöser",
-        "CameraControl.Settings.ShowLiveBadge" : "Zeige Live-Abzeichen",
+        "CameraControl.Settings.ShowLiveBadge" : "Zeige Live-Badge",
```

It was fine with Abzeichen before, Badge is the English word for it.
```suggestion
        "CameraControl.Settings.ShowLiveBadge" : "Zeige Live-Abzeichen",
```